src/coreComponents/constitutive/fluid/multifluid/CO2Brine/functions/BrineEnthalpy.cpp

```diff
@@ -73,21 +73,34 @@ real64 michaelidesBrineEnthalpy( real64 const & T,
 
 void calculateBrineEnthalpy( PTTableCoordinates const & tableCoords,
                              real64 const & m,
-                             array1d< real64 > const & enthalpies )
+                             array1d< real64 > const & enthalpies,
+                             bool const printTable )
 {
 
   localIndex const nTemperatures = tableCoords.nTemperatures();
 
+  std::ofstream table_file;
+  if( printTable )
+  {
+    table_file.open( "BrineEnthalpy.csv" );
+    table_file << "T[C],Enthalpy" << std::endl;
```

Comment:
Consider using `"\n"` instead of `std::endl`, since there is no need to flush the stream after every line

Reply:
no one:

Sergey: no need to flush